pkg/security/podsecuritypolicy/sysctl/mustmatchpatterns_test.go

```diff
@@ -89,7 +89,7 @@ func TestValidate(t *testing.T) {
 		testDisallowed := func(key string, category string) {
 			for _, s := range v.disallowed {
 				pod.Annotations = map[string]string{
-					key: api.PodAnnotationsFromSysctls([]api.Sysctl{{s, "dummy"}}),
+					key: api.PodAnnotationsFromSysctls([]api.Sysctl{{Name: s, Value: "dummy"}}),
```

Comment:
Clearly this is an improvement but is it really necessary for go1.7? (same comment in other places, too)